odoo/tools/populate.py

```diff
@@ -93,7 +91,7 @@ def iterate(vals, weights=None, seed=False, formatter=format_str, then=None):
     """
     def generate(iterator, field_name, model_name):
         counter = 0
-        for val in vals: # iteratable order is important, shortest first
+        for val in vals:  # iteratable order is important, shortest first
```

Comment:
This is useless diff for example. Same with removing the import, if there is no benefit, just leave them, it will help people checking your commit in two years to figure out what you were doing. A patch should be as minimal as possible, it will improve readability. Only the `return` clause should be in the diff of this file.

Reply:
Ok I will remove the cleaning.

Reply:
It is done!